internal/cmd/nexec.go

```diff
@@ -147,32 +149,49 @@ func execInClusterNodes(args []string) error {
 	// In distributed shell mode, TTY is forced to be disabled
 	nodeExecNoTty = true
 
+	// In case nothing is configured, all nodes will be executed concurrently
+	if nodeExecMaxParallel <= 0 {
```

Comment:
should you consider the scenario when `nodeExecMaxParallel` is higher than `len(nodes)`?